packages/uswds-utilities/src/styles/rules/add-aspect.scss

```
@@ -3,16 +3,17 @@
 ASPECT
 ----------------------------------------
 usage:
-  .aspect-[value]
+  .add-aspect-[key]
 ----------------------------------------
 output:
-  padding-left: 0;
-  padding-right: 0;
-  padding-top: 0;
-  padding-bottom: 100%;
+  aspect-ratio: [value];
 ----------------------------------------
 example:
-  .aspect-16x9
+  .add-aspect-16x9 {
+    height: inherit;
+    padding: inherit;
+    aspect-ratio: 1.7777777778;
+    max-width: 100%; }
```

Love the improved comments.

Small change to indent to match standards below.

```suggestion
    max-width: 100%; 
  }
```

---

This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)